conf/distro/nilrt.conf

```diff
@@ -25,3 +25,23 @@ KERNEL_IMAGEDEST = "boot/runmode"
 ROOT_HOME = "/home/admin"
 
 IMAGE_LINGUAS ?= "en-us en-us.iso-8859-1"
+
+BUILD_IMAGES_FROM_FEEDS = "1"
+
+# Get NIOE packages from local deploy feed and NI packages from web feed
+IPK_NIOE_SUBFEED_URI ?= "file://${DEPLOY_DIR_IPK}"
+
+FEED_SRC ?= "download.ni.com/ni-linux-rt/feeds"
+NI_MAIN_FEED_VERSION ?= "2020.5"
+NI_SALT_FEED_VERSION ?= "20.0"
+IPK_NI_MAIN_SUBFEED_URI ?= "http://${FEED_SRC}/${NI_MAIN_FEED_VERSION}/ni-main/"
+IPK_NI_SALT_SUBFEED_URI ?= "http://${FEED_SRC}/SystemLink/salt/${NI_SALT_FEED_VERSION}/${MACHINE}/${TUNE_PKGARCH}/"
```

Comment:
I think we can drop the `NI_SALT` content here. Those salt feeds shouldn't contain anything that isn't already in the NIOE feeds. If they do, we need to reconcile the differences.